build.gradle.kts

```diff
@@ -8,7 +8,14 @@ configurations {
 }
 
 dependencies {
+  // Spring boot dependencies
   implementation("org.springframework.boot:spring-boot-starter-webflux")
+  implementation("uk.gov.justice.service.hmpps:hmpps-kotlin-spring-boot-starter:0.2.4")
```

Comment:
The key thing to be aware of here is this shared common component pulls in the necessary config for auth setup.  We don't need a dedicated class (it also gives us a entry point to providing SAR data when the time comes).

We will need one for our WebClient configs though.